### ISSUE TITLE
Implement content search function from Everything

### ISSUE THREAD

**Issue by linuszh** [Wox-launcher/Wox#2905]
**Is your feature request related to a problem? Please describe.**
I'm using the "content:" feature from everything a lot and i can't seem to make it work in Wox. It's very handy when you looking for something specific and you forgot the file name.  

**Describe the solution you'd like**
Be able to open the Wox and type content: ****** to search for the string ****** in text files.

**Describe alternatives you've considered**
Basically the only alternative for me right now is to open everything and do the query there.

**Comment by bao-qian** [comment]
this should be support.
could you give an concrete query example with screenshot for wox and everything?

**Comment by bao-qian** [comment]
close due to no further activity, reopen if you still have problem
